transport/http/outbound.go

```diff
@@ -282,7 +282,7 @@ func (o *Outbound) callWithPeer(
 		// maintenance loop resumes probing for availability.
 		p.OnDisconnected()
 
-		return nil, err
+		return nil, yarpcerrors.UnknownErrorf("unknown error from http client: %s", err.Error())
```

Comment:
You’re correct. We might be able to classify some of these errors as Unavailable, but Unknown is a safe foundation.